lib/ansible/module_utils/ec2.py

```diff
@@ -554,6 +554,12 @@ def _hashable_policy(policy, policy_list):
          ('Version', (u'2012-10-17',)))]
 
     """
+    # Amazon will automatically convert bool and int to strings for us
+    if isinstance(policy, bool):
+        return tuple([str(policy).lower()])
```

Comment:
I'm confused by the inconsistency of these two return values.
* Why are we returning tuples? Is there a reason not to just return `str(policy).lower()` and `str(policy)` ?
* I can see why the `bool` result is wrapped in a list before casting to tuple (to avoid `('t', 'r', 'u', 'e')` being the result) - why not wrap the `int` result (otherwise we get `('1', '2', '3')`)

Reply:
1) I'm returning a tuple because strings are currently returned wrapped in a tuple I believe it's done that way so that a single string is evaluated the same ways as an array containing one string.

2) Because my test case was a single digit in so I missed that.